### PR TITLE
List of repos under start-work/new PR/request review doesn't pick up newly added ones w/o a reload

### PR DESCRIPTION
**Pull request by ppezaris**

### CODE REVIEW

**Comment by ppezaris** [comment]
this posts directly to github

**Comment by ppezaris** [comment]
Is this ready to be merged in?

**Comment by bcanzanella** [comment]
> 
> 
> Is this ready to be merged in?

is this a legit PR? those changes got into develop on Sept. 1

**Comment by bcanzanella** [comment]
here's a comment

File: shared/agent/src/git/repositories.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import * as chokidar from "chokidar";
 import * as fs from "fs";
 import * as path from "path";
-import { CommitsChangedData } from "protocol/agent.protocol";
+import { CommitsChangedData, WorkspaceChangedData } from "protocol/agent.protocol";
```
Comment (bcanzanella) [inline shared/agent/src/git/repositories.ts:5]: line 5!

**Pull request closed without merging**